bpf/lib/common.h

```diff
@@ -524,13 +524,14 @@ enum {
 
 /* Service flags (lb{4,6}_service->flags) */
 enum {
-	SVC_FLAG_EXTERNAL_IP  = (1 << 0),  /* External IPs */
-	SVC_FLAG_NODEPORT     = (1 << 1),  /* NodePort service */
-	SVC_FLAG_LOCAL_SCOPE  = (1 << 2),  /* externalTrafficPolicy=Local */
-	SVC_FLAG_HOSTPORT     = (1 << 3),  /* hostPort forwarding */
-	SVC_FLAG_AFFINITY     = (1 << 4),  /* sessionAffinity=clientIP */
-	SVC_FLAG_LOADBALANCER = (1 << 5),  /* LoadBalancer service */
-	SVC_FLAG_ROUTABLE     = (1 << 6),  /* Not a surrogate/ClusterIP entry */
+	SVC_FLAG_EXTERNAL_IP   = (1 << 0),  /* External IPs */
+	SVC_FLAG_NODEPORT      = (1 << 1),  /* NodePort service */
+	SVC_FLAG_LOCAL_SCOPE   = (1 << 2),  /* externalTrafficPolicy=Local */
+	SVC_FLAG_HOSTPORT      = (1 << 3),  /* hostPort forwarding */
+	SVC_FLAG_AFFINITY      = (1 << 4),  /* sessionAffinity=clientIP */
+	SVC_FLAG_LOADBALANCER  = (1 << 5),  /* LoadBalancer service */
+	SVC_FLAG_ROUTABLE      = (1 << 6),  /* Not a surrogate/ClusterIP entry */
+	SVC_FLAG_LOCALREDIRECT = (1 << 7),  /* local redirect */
```

Comment:
This is mismatched with userspace. Userspace defines another flag at `(1 << 7)`, `serviceFlagSourceRange` and defines this new flag as `(1 << 8)`.

Reply:
Good eye! Thanks, I'll fix it.